src/Test/FixCommonErrorsTest.cs

```diff
@@ -756,6 +761,8 @@ public void FixUnneededSpaces3()
             }
         }
 
+        [TestMethod]
+        [DeploymentItem("SubtitleEdit.exe")]

```

Comment:
Deployment attributes removed should I push it on top of this commit?! // @niksedk